backend/onyx/document_index/vespa/shared_utils/vespa_request_builders.py

```diff
@@ -106,3 +107,25 @@ def build_vespa_id_based_retrieval_yql(
 
     id_based_retrieval_yql_section += ")"
     return id_based_retrieval_yql_section
+
+
+def build_deletion_selection(doc_id: str, version_cutoff: int, doc_type: str) -> str:
```

Comment:
Any way to test this?  Or should we put this in a follow on PR?

Reply:
There certainly is! If you query vespa for a specific number of docs, pull in updates to a connector (which now has fewer chunks), you should see fewer results for the doc ID when you query. Tested this one manually but could probably use a test!

Reply:
There certainly is! If you query vespa for a specific number of docs, pull in updates to a connector (which now has fewer chunks), you should see fewer results for the doc ID when you query. Tested this one manually but could probably use a test!

Reply:
ok, let's add a linear issue to follow up